moj-node/assets/sass/components/_featured-content-item.scss

```diff
@@ -52,6 +53,27 @@ $mauve: #6f72af;
     }
   }
 }
+.govuk-hub-featured-link:focus .govuk-hub-featured-content-item {
+  -webkit-transform: scale(1.1);
+  -ms-transform: scale(1.1);
+  transform: scale(1.1);
+  z-index: 100;
+  -webkit-box-shadow: 0px 0px 20px 5px rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 0px 0px 20px 5px rgba(0, 0, 0, 0.75);
+  box-shadow: 0px 0px 20px 5px rgba(0, 0, 0, 0.75);
+
+  .govuk-hub-featured-content-item__overlay {
```

Comment:
I know it's an unpopular opinion but can we avoid nesting?
It makes it hard to grep for css.
Let's use sass for mixins and variables only?

Reply:
Copied from the section above, so exactly the same nesting stuff, am happy to refactor these two bits if you like?

Reply:
I know.. that was pre me lol.. But yeah if you can do that that would be great

Reply:
Should be sorted, have only done the part I've reused though as there's a fair amount of nested stuff around :)